packages/framework/ai-collab/src/test/ai-collab/debugLog.spec.ts

```diff
@@ -0,0 +1,423 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+
+// eslint-disable-next-line import/no-internal-modules
+import { createIdCompressor } from "@fluidframework/id-compressor/internal";
+// eslint-disable-next-line import/no-internal-modules
+import { MockFluidDataStoreRuntime } from "@fluidframework/test-runtime-utils/internal";
+import {
+	SharedTree,
+	SchemaFactory,
+	TreeViewConfiguration,
+	type TreeView,
+	type ITree,
+	// eslint-disable-next-line import/no-internal-modules
+} from "@fluidframework/tree/internal";
+import { OpenAI } from "openai";
+
+import { aiCollab } from "../../aiCollab.js";
+import type {
+	AiCollabErrorResponse,
+	AiCollabSuccessResponse,
+	DebugEvent,
+	EventFlowDebugEvent,
+} from "../../aiCollabApi.js";
+import {
+	type ApplyEditFailure,
+	type ApplyEditSuccess,
+	type CoreEventLoopStarted,
+	type CoreEventLoopCompleted,
+	type FinalReviewCompleted,
+	type FinalReviewStarted,
+	type GenerateTreeEditCompleted,
+	type GenerateTreeEditStarted,
+	type LlmApiCallDebugEvent,
+	type PlanningPromptCompleted,
+	type PlanningPromptStarted,
+	EventFlowDebugNames,
+	// eslint-disable-next-line import/no-internal-modules
+} from "../../explicit-strategy/debugEvents.js";
+
+const sf = new SchemaFactory("TestApp");
+class TestAppSchema extends sf.object("TestAppSchema", {
+	title: sf.string,
+	tasks: sf.array(
+		sf.object("Task", {
+			title: sf.string,
+			description: sf.string,
+		}),
+	),
+}) {}
```

Comment:
I think a recent change by Craig is already merged, which made all the schemas available as static members of `SchemaFactory`, so we don't need to instantiate it anymore and can just do `SchemaFactory.string`, etc.